include/circt/Dialect/SV/SVExpressions.td

```diff
@@ -186,3 +186,29 @@ def SampledOp : SVOp<"system.sampled", [SameOperandsAndResultType]> {
   let results = (outs AnyType:$sampledValue);
   let assemblyFormat = "$expression attr-dict `:` qualified(type($expression))";
 }
+
+/// Handle to stdout.
+def StdOut: I32EnumAttrCase<"StdOut", 1, "stdout">;
+/// Handle to stderr.
+def StdErr: I32EnumAttrCase<"StdErr", 2, "stderr">;
```

Comment:
Would it make sense to also have `stdin` here?

Reply:
I think it can be easily added once there is a mechanism to read from descriptors.